src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger.java

```diff
@@ -769,7 +769,7 @@ public ListBoxModel doFillVerdictCategoryItems() {
      *
      * @return the store of running jobs.
      */
-    /*package*/ synchronized RunningJobs getRunningJobs() {
+    /*package*/ RunningJobs getRunningJobs() {
         if (runningJobs == null) {
             runningJobs = new RunningJobs();
```

Comment:
Removing the synchronized here means there is risk of dirty write.

Reply:
`synchronized` was not needed, `runningJobs` was supposed to be initialised during class creation, but it was not final. So, I made it final and removed the check.